modules/gke-cluster-autopilot/variables.tf

```diff
@@ -216,9 +216,34 @@ variable "service_account" {
 }
 
 variable "tags" {
-  description = "Network tags applied to nodes."
+  description = "Applies the specified network tags to all nodes in this Autopilot cluster. Network tags are metadata on Compute Engine virtual machines (VMs) that allow you to make firewall rules and routes applicable to specific VM instances. In GKE, you can use network tags to make firewall rules or routes applicable to the nodes in your cluster."
```

Comment:
This description is too long, we assume people know what the attributes are and only expand when the type might not be obvious.

Reply:
OK, will rollback 🙂

Reply:
Description was rolled back as requested. Item resolved 🎉